solr/solrj/src/java/org/apache/solr/client/solrj/request/beans/CreateCorePayload.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.solr.client.solrj.request.beans;
+
+import org.apache.solr.common.annotation.JsonProperty;
+import org.apache.solr.common.util.ReflectMapWriter;
+
+import java.util.List;
+import java.util.Map;
+
+public class CreateCorePayload implements ReflectMapWriter {
+    @JsonProperty(required = true)
+    public String name;
+
+    @JsonProperty
+    public String instanceDir;
+
+    @JsonProperty
+    public String dataDir;
+
+    @JsonProperty
+    public String ulogDir;
+
+    @JsonProperty
+    public String schema;
+
+    @JsonProperty
+    public String config;
+
+    @JsonProperty
+    public String configSet;
+
+    @JsonProperty
+    public Boolean loadOnStartup;
+
+    // If our JsonProperty clone was more feature-rich here we could specify the property be called 'transient', but
```

Comment:
should we just pick a different parameter name and not sue the word transient?

Reply:
Well that's what I am doing here for the v2 API.  The property name in v2 is "isTransient".

The v1 query parameter is still named "transient".  And we could change that, but it'd run afoul of backwards compatibility concerns.  I'd need an 8.x release to sneak a deprecation into, which doesn't seem super likely at this point.